hm_1/tasks/task_01_02.py

```diff
@@ -0,0 +1,19 @@
+"""
+Given a cell with "it's a fib sequence" from slideshow,
+    please write function "check_fib", which accepts a Sequence of integers, and
+    returns if the given sequence is a Fibonacci sequence
+We guarantee, that the given sequence contain >= 0 integers inside.
+"""
+from typing import Sequence
+
+
+def check_fibonacci(data: Sequence[int]) -> bool:
+    if data == [0]:
+        return True
+    if len(data) <= 1 or (data[0:2] != [0, 1]):
```

Comment:
`len(data) <= 1`  Does this check make sense now?

Reply:
Yes, No sense in that check, I'll delete it.